app/views/editor/rich.html.erb

```diff
@@ -314,6 +314,7 @@
       <% if @lat and @lon %>
```

Comment:
Ah, actually perhaps we need a little more logic here? Inside this `if`, maybe we need a second `if @zoom`?

Otherwise what if zoom is not included but lat and lon are?

Thank you!!!

Reply:
Oh yes! We need that logic 👍

Reply:
OK, I amended this, I think this is good to go now!